discovery_engine_flutter/example/ios/Podfile

```diff
@@ -37,11 +37,8 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
-    # TODO: check if we need to add this when we add binaries
-    # target.build_configurations.each do |config|
-    #   config.build_settings['ENABLE_BITCODE'] = 'NO'
```

Comment:
These commented-out settings, do we know why they were once used? I'm wondering if omitting them may e.g. cause problems on a device (as opposed to the simulator).